<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game2</class>
 <widget class="QMainWindow" name="Game2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Button_EndGame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>End Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Submit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_Participants">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_UserName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
